native/packages/react-native-bpk-component-icon/BpkSvgsFont.podspec

```diff
@@ -0,0 +1,16 @@
+require 'json'
+version = JSON.parse(File.read('package.json'))["version"]
+
+Pod::Spec.new do |s|
+
+  s.name           = "BpkSvgsFont"
+  s.version        = version
+  s.summary        = "Backpack's collection of SVGs."
+  s.homepage       = "https://github.com/Skyscanner/backpack/tree/master/packages/react-native-bpk-component-icon"
+  s.license        = "Apache License 2.0"
+  s.author         = { "Backpack" => "backpack@skyscanner.net" }
+  s.resources      = "./node_modules/bpk-svgs/dist/font/*.ttf"
```

Comment:
I might be wrong, but the way I saw this working is that the scripts will point to the `.ttf`s in the`react-native-bpk-component-icon` package, and the icon package will have a dependency on a specific version of `bpk-svgs`

Reply:
This seems correct given that it does work :)

Reply:
Note: this _could_ break if consumers install a _different_ version of `bpk-svgs` than the one depended on by `react-native-bpk-component-icon` because npm will hoist the top level dependency to that path over the dependency specified by the icon component. E.g.:
```
node_modules/
    react-native-bpk-component-icon/
    bpk-svgs/
```

User runs `npm install bpk-svgs --save-dev`, which results in:
```
node_modules/
    react-native-bpk-component-icon/
        node_modules/
            bpk-svgs/
    bpk-svgs/ // <- our podspec now points to the newer version which could be breaking
```

Reply:
not sure we can do anything about it, although the icon component is importing both the font and the mapping, so even if an old version is added as devDeps the icon component will not "break"

Reply:
Yeah I was concerned about the implications of this too. I guess we'll see how it behaves in the wild

Reply:
No you are correct the icon component will be fine - just the rendering might be affected because the app manifest points to the wrong `.ttf`. I reckon it's a low risk - why would consumers depend on `bpk-svgs` directly?